<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateFood</class>
 <widget class="QDialog" name="updateFood">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="promptLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>297</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="editGroup">
    <item>
     <widget class="QDoubleSpinBox" name="newPrice">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <double>9999999.000000000000000</double>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonGroup">
      <item>
       <widget class="QPushButton" name="confirmButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
